src/config.ts

```diff
@@ -352,13 +352,13 @@ export function configRead(): IConfig {
     "--pantalaimon-password-path"
   );
   if (explicitAccessTokenPath !== undefined) {
-    config.accessToken = fs.readFileSync(explicitAccessTokenPath, "utf8");
+    config.accessToken = fs.readFileSync(explicitAccessTokenPath, "utf8").trim();
```

Comment:
Would it be more appropriate to just extract the first line from the file?

Reply:
I'm not sure, this also deals with accidental leading newlines (ie. scripts), or also accidental spaces for whatever reason.

Accounting for all of those, you'd need to iterate lines, find the one with the token, trim spaces.

Reply:
(TL;DR this handles a lot more edge cases than just extracting the first line)

Reply:
We shouldn't be handling accidental spaces or newlines though, we should explode if there are those.